### DISCUSSION TITLE
Bulk download

### DISCUSSION THREAD

**Discussion by TGuhDYkq** [EchterAlsFake/Porn_Fetch#61] [Q&A]
I use the CLI version. There I can e.g. B. store a PornHub list. After the scan, the CLI asks which files I want to download.
What do I have to enter to download all videos from the list?

**Comment by EchterAlsFake** [comment]
Hi, I somehow forgot to implement such option to just download all videos without entering all numbers. I'll implement it.
Thanks for the reminder xD

**Reply by griffinprince009** [reply]
Good day , I use the GUI version , and just would like to know if there is a way to download all videos from a channel or shorcut ?
I tried looking but cant seem to get it right ?

**Reply by EchterAlsFake** [reply]
Nope, this is not possible right now. This was possible in version 3.8, but I removed the feature due to design changes in the UI, but I am already on re-implementing this.
